SpectrometerHDF5OutputFile.cpp

```diff
@@ -3395,10 +3405,47 @@ void cSpectrometerHDF5OutputFile::addNoiseDiode6_7GHzPWMFrequency(int64_t i64Tim
 
 void cSpectrometerHDF5OutputFile::addSourceSelection(int64_t i64Timestamp_us, const string &strSourceName, const string &strStatus)
 {
+    // Example Source Name string: VIRGOA, radec, 12:30:49.400000, 12:23:28.000000
```

Comment:
Is this a katpoint string? You're skipping the `radec` token, so if you ever get passed an azel, this will just silently and unknowingly record incorrect values?

Reply:
Have updated - coordinate field added in next commit.